web/src/main/java/org/cbioportal/web/SessionServiceController.java

```diff
@@ -93,8 +103,11 @@ private PageSettings getRecentlyUpdatePageSettings(String query) {
         return sessions.isEmpty() ? null : sessions.get(0);
     }
 
-    private ResponseEntity<Session> addSession(SessionType type, Optional<SessionOperation> operation,
-            JSONObject body) {
+    private ResponseEntity<Session> addSession(
+        SessionType type, 
+        Optional<SessionOperation> operation,
```

Comment:
Ok, I see, this is just formatting. I was going to complain about using an optional where `@Nullable` would suffice, but this isn't your fault.